SDP: fix compilation warnings

Fix the following compilation warning:
drivers/infiniband/ulp/sdp/sdp_main.c:3133: warning: initialization from incompatible pointer type
drivers/infiniband/ulp/sdp/sdp_main.c:3140: warning: initialization from incompatible pointer type

Signed-off-by: Dotan Barak <dotanb@dev.mellanox.co.il>
---

 drivers/infiniband/ulp/sdp/sdp_main.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/infiniband/ulp/sdp/sdp_main.c b/drivers/infiniband/ulp/sdp/sdp_main.c
index 374e3a9..a2e1a68 100644
--- a/drivers/infiniband/ulp/sdp/sdp_main.c
+++ b/drivers/infiniband/ulp/sdp/sdp_main.c
@@ -2989,13 +2989,13 @@ static int sdp_create_ipvx_socket(struct net *net, struct socket *sock, int prot
 }
 
 #if defined(CONFIG_IPV6) || defined(CONFIG_IPV6_MODULE)
-static int sdp_create_v6_socket(struct net *net, struct socket *sock, int protocol)
+static int sdp_create_v6_socket(struct net *net, struct socket *sock, int protocol, int kern)
 {
 	return sdp_create_ipvx_socket(net, sock, protocol, &sdp_ipv6_proto_ops);
 }
 #endif
 
-static int sdp_create_v4_socket(struct net *net, struct socket *sock, int protocol)
+static int sdp_create_v4_socket(struct net *net, struct socket *sock, int protocol, int kern)
 {
 	return sdp_create_ipvx_socket(net, sock, protocol, &sdp_ipv4_proto_ops);
 }
